Andrea / exercise-1/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pagina prueba</title>
+    <link rel="stylesheet" href="stylesheet.css"
+</head>
+<body>
+    <header>
+       <nav class="top-menu">
+        <img class="logo" src="logo.png" alt="imagen">
+           <ul class="top-menu__links">
+               <li>
+                   <a href="#">features</a>
+               </li>
+               <li>
+                   <a href="#">templates</a>
+               </li>
+               <li>
+                   <a href="#">integrations</a>
+               </li>
+               <li>
+                   <a href="#">pricing</a>
+               </li>
+               <!-- Botones -->
+               <li>
+                   <a class="top-menu__button" href="#">sign in</a>
+               </li>
+               <li>
+                   <a class="top-menu__button" href="#">sign up</a>
+               </li>
+           </ul>
+       </nav>
+
+        <!-- Greetings -->
+        <section class="hero">
+            <h1>The magic of reading</h1>
+        </section>
+    </header>
+
+
+    <!-- Achievements -->
+    <section>
+        <ul class="section-menu">
+            <li>
+                <p><img src="icon.png" alt="icon">genres</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">buy</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">fanart</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">communities</p>
+            </li>
+    </section>
+
+    <main>
```

Comment:
I would make the section that contains the list of Achievements part of the main so it can be included by the screenreaders inside the `main landmark`. Landmarks are sections defined in your page that allow screen readers to organize the content in a website in sections that can be quickly traversed. More info at the bottom of this page https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/aria-landmarks